nimble/host/src/ble_att_svr.c

```diff
@@ -1360,7 +1360,7 @@ ble_att_svr_build_read_type_rsp(uint16_t conn_handle,
         *att_err = 0;
 
         /* Fill the response base. */
-        rsp->batp_length = htole16(sizeof(*data) + prev_attr_len);
```

Comment:
Hi, I'm not sure if I get this one on how it would fix anything, could you explain it a bit?

Reply:
`batp_length` type is `uint8_t`, so calling `htole16` basically discards the value on a big-endian system.

Reply:
ahh right!